src/repos/ArticleRepo.ts

```diff
@@ -1,4 +1,5 @@
 import Filter from 'bad-words';
+import moment from 'moment';
```

Comment:
Seems like moment is deprecated, can we not do something like `new Date().getDate() - 30` instead?

https://momentjs.com/docs/

Reply:
It is deprecated but its still a pretty popular and widely used library thats easy for date time usage as suggested @grebneerg what do you think?

Reply:
I was unaware that moment has been deprecated. My instinct would be not to use it, especially given that they aren't going to add immutability and recommend libraries that do. [Luxon](https://moment.github.io/luxon/) seems to be the literal successor, hosted under the same organization on github, and looks very nice, but [day.js](https://day.js.org/) looks pretty good as well. I would still stay away from using built in JS dates directly, since `Date` objects are mutable and parsing and timezone handling really isn't great.